Sidebar content

dodecadumpling opened the pull request:
id's 1 to 7 have photos in them.
I have also included the foundation stone image Maria forwarded.
Others id photos to follow later ....

Drewbi committed:
Add image support

Drewbi committed:
Info fix

Drewbi committed:
Merge branch 'master' of https://github.com/Drewbi/buildings-in-landscape-uwa into drewbi/sidebar-content

Drewbi committed:
Fix marker spot missing problem

Drewbi committed:
Remove image margin

Commit:
2d photos up to id=5

Commit:
images to id=5

Commit:
resolved merge conflicts

Commit:
2D images up to id=14. Added also Reflection pool image as bonus.

Drewbi committed:
Fix sidebar scroll

Drewbi committed:
Style caption

Drewbi committed:
Merge branch 'sidebar-content' of https://github.com/Drewbi/buildings-in-landscape-uwa into sidebar-content

Drewbi committed:
Update padding

Drewbi commented on src/data/info.json:
```diff
@@ -62,12 +139,24 @@
     "title": "Whitfeld Court",
-    "text": "In 1927, Leslie Wilkinson revised the original campus plan and established the fundamentals of the modern campus. The plan reoriented the campus to the north – south alignment that dominates today. This spine established the three major open spaces:  The Court of Honour named by Alsop (Whitfeld Court), The Great Court, and James Oval. Dr W Somerville promoted the concept that the are ...
+    "text": "In 1927, Leslie Wilkinson revised the original campus plan and established the fundamentals of the modern campus. The plan reoriented the campus to the north – south alignment that dominates today. This spine established the three major open spaces:  The Court of Honour named by Alsop (Whitfeld Court), The Great Court, and James Oval. Dr W Somerville promoted the concept that the are ...
+    "images": [
```
I'm not sure about using this image here, We'll ask fran

dodecadumpling replied:
If that was the picture of Whitfeld statue bust, that was a placeholder. 

I have corrected it with the latest commit. Sorry about that.

Drewbi replied:
Ah good good

Commit:
up to id=18 now.

Commit:
up to id=19. AlSO fixed image of Whitefeld court
Merge branch 'sidebar-content' of https://github.com/Drewbi/buildings-in-landscape-uwa into sidebar-content

Commit:
2D photos are in place. id = 9,10,23 need to be checked with client AND whether years should be preceded by circa (where applicable)

Drewbi reviewed (approved):
Great work, really brings it together having images
